public_html/wp-content/mu-plugins/blocks/includes/components/post-list.php

```diff
@@ -43,7 +43,7 @@ function render_post_list( array $rendered_items, $layout = 'list', $columns = 1
 	?>
 		<ul class="<?php echo esc_attr( $container_classes ); ?>">
 			<?php foreach ( $rendered_items as $item ) : ?>
-				<li class="wordcamp-post-list-item wordcamp-clearfix">
+				<li class="wordcamp-post-list__post wordcamp-clearfix">
```

Comment:
Hmm, so right now, root has actually `wordcamp-block wordcamp-post-list layout-list wordcamp-organizers__posts` and then children `li` has `wordcamp-post-list__post`, and whose children has `wordcamp-organizers__post has-slug-organizer-1`, which sound reasonable to me, because `li` is rendered by the `post-list` component, and then `wordcamp-organizers__post has-slug-{organizer-slug}` which is a `div` is rendered by the block itself.

Are you suggesting that we merge the classes in `li` and `div`? Looks like we can do that by passing the class name while rendering `post-list`. 
For the root element, I agree that `wordcamp-organizers` is cleaner that `wordcamp-organizers__posts`, but then because of the way `js` is structured, it won't follow the guideline of only treating `index.js` export as root and rest of all as descendants. What do you think?

Reply:
> Hmm, so right now, root has actually wordcamp-block wordcamp-post-list layout-list wordcamp-organizers__posts and then children li has wordcamp-post-list__post, and whose children has wordcamp-organizers__post has-slug-organizer-1, which sound reasonable to me, because li is rendered by the post-list component, and then wordcamp-organizer wordcamp-organizers__post has-slug-{organizer-slug} which is a div is rendered by the block itself.

Ah, good point 👍 

> Are you suggesting that we merge the classes in li and div? Looks like we can do that by passing the class name while rendering post-list.

🤔 yeah, I think so. It does feel weird that we have the `li` and then the `div`; the `div` feels unnecessary.

It seems like we should replace the `<div>` with a `Fragment`, and pass the class names for the `li` like you said.

I don't feel too strongly, though, so let me know if you see any problems with that.

> For the root element, I agree that wordcamp-organizers is cleaner that wordcamp-organizers__posts, but then because of the way js is structured, it won't follow the guideline of only treating index.js export as root and rest of all as descendants. What do you think?

Yeah, I see your point. It feels like the convention kind of breaks down here, and is complicated by the fact that a block is more than a simple component, and it has the "index" which is really just a controller, and the "edit" which is the view.

It seems like maybe `edit.js` should have a `<div class="wordcamp-organizers">` instead of a `<Fragment>`, but 🤷‍♂ . Trying to follow this convention is getting pretty awful :(

Reply:
Noting here that in separate discussion, we agreed to
1. Make root class name as `wordcamp-organizer` i.e. to treat `block-content.js` as the root file instead of `index.js`
2. Keep both `li` and `div` as it is now.

Reply:
Done in https://github.com/WordPress/wordcamp.org/pull/123/commits/af6f25b4a21ebb99b8077bddf8fe6e8ead8a0087